Build command run build-managed -packages fails with error MSB1009

There is an error in implementation of run build-managed -packages command which according to displayed run -? help should build output packages. Command when run displays following output:
```
e:\src\ms\dotnet\standard>run build-managed -packages
Tools are already initialized.
Running: "e:\src\ms\dotnet\standard\Tools\dotnetcli\dotnet.exe" "e:\src\ms\dotnet\standard\Tools\run.exe" build-managed -packages
Running: e:\src\ms\dotnet\standard\Tools\msbuild.cmd /nologo /v:minimal /flp:v=normal /clp:Summary /maxcpucount /nodeReuse:false /l:BinClashLogger,Tools\net46\Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log /p:FilterToOSGroup=win7 /p:ConfigurationGroup=Debug  /flp:v=normal;LogFile=build-packages.log  /flp2:warningsonly;logfile=msbuild.wrn  /flp3:errorsonly;logfile=msbuild.err  netstandard/package.builds
MSBUILD : error MSB1009: Project file does not exist.
Switch: netstandard/package.builds
Command execution failed with exit code 1.
```

Error is caused by invalid command argument set in config.json file. The argument points to non existing `netstandard/package.builds` project. After changing argument to `packages.builds` command correctly invokes MSBuild and builds output packages. This time output of the command is as follows:
```
e:\src\ms\dotnet\standard>run build-managed -packages
Tools are already initialized.
Running: "e:\src\ms\dotnet\standard\Tools\dotnetcli\dotnet.exe" "e:\src\ms\dotnet\standard\Tools\run.exe" build-managed -packages
Running: e:\src\ms\dotnet\standard\Tools\msbuild.cmd /nologo /v:minimal /flp:v=normal /clp:Summary /maxcpucount /nodeReuse:false /l:BinClashLogger,Tools\net46\Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log /p:FilterToOSGroup=win7 /p:ConfigurationGroup=Debug  /flp:v=normal;LogFile=build-packages.log  /flp2:warningsonly;logfile=msbuild.wrn  /flp3:errorsonly;logfile=msbuild.err  packages.builds
  netstandard -> e:\src\ms\dotnet\standard\bin\ref\netstandard\2.0.0.0\netstandard.dll
  System.Runtime.Serialization.Json -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Serialization.Json.dll
  System.Threading.ThreadPool -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.ThreadPool.dll
  System.Xml.ReaderWriter -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.ReaderWriter.dll
  System.Security.Principal -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Principal.dll
  System.Security.SecureString -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.SecureString.dll
  System.Runtime.Serialization.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Serialization.Primitives.dll
  System.Xml.XPath -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.XPath.dll
  System.ValueTuple -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ValueTuple.dll
  System.Xml.XmlSerializer -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.XmlSerializer.dll
  System.Xml.XPath.XDocument -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.XPath.XDocument.dll
  System.Threading.Timer -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.Timer.dll
  System.Threading -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.dll
  System.Text.RegularExpressions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Text.RegularExpressions.dll
  System.Threading.Tasks.Parallel -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.Tasks.Parallel.dll
  System.Xml.XDocument -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.XDocument.dll
  System.Threading.Overlapped -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.Overlapped.dll
  System.Security.Cryptography.X509Certificates -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Cryptography.X509Certificates.dll
  System.Threading.Thread -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.Thread.dll
  System.Threading.Tasks -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Threading.Tasks.dll
  System.Runtime.Handles -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Handles.dll
  System.Text.Encoding -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Text.Encoding.dll
  System.Security.Cryptography.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Cryptography.Primitives.dll
  System.Security.Claims -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Claims.dll
  System.Reflection.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Reflection.Primitives.dll
  System.Security.Cryptography.Algorithms -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Cryptography.Algorithms.dll
  System.Text.Encoding.Extensions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Text.Encoding.Extensions.dll
  System.Security.Cryptography.Csp -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Cryptography.Csp.dll
  System.Security.Cryptography.Encoding -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Security.Cryptography.Encoding.dll
  System.Runtime.Numerics -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Numerics.dll
  System.Runtime.InteropServices -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.InteropServices.dll
  System.Net.WebHeaderCollection -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.WebHeaderCollection.dll
  System.Xml.XmlDocument -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Xml.XmlDocument.dll
  System.Runtime.CompilerServices.VisualC -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.CompilerServices.VisualC.dll
  System.Runtime -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.dll
  System.Net.NetworkInformation -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.NetworkInformation.dll
  System.Runtime.Serialization.Xml -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Serialization.Xml.dll
  System.Resources.Writer -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Resources.Writer.dll
  System.IO.UnmanagedMemoryStream -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.UnmanagedMemoryStream.dll
  System.Net.WebSockets -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.WebSockets.dll
  System.Runtime.Serialization.Formatters -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Serialization.Formatters.dll
  System.Runtime.InteropServices.RuntimeInformation -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.InteropServices.RuntimeInformation.dll
  System.Reflection.Extensions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Reflection.Extensions.dll
  System.IO.FileSystem.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.FileSystem.Primitives.dll
  System.Net.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Primitives.dll
  System.Reflection -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Reflection.dll
  System.Resources.Reader -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Resources.Reader.dll
  System.Net.Sockets -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Sockets.dll
  System.Runtime.Extensions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Runtime.Extensions.dll
  System.Linq.Parallel -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Linq.Parallel.dll
  System.IO.Compression -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.Compression.dll
  System.Net.Security -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Security.dll
  System.ObjectModel -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ObjectModel.dll
  System.Resources.ResourceManager -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Resources.ResourceManager.dll
  System.IO.Pipes -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.Pipes.dll
  System.Net.Http -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Http.dll
  System.Drawing.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Drawing.Primitives.dll
  System.Net.Requests -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Requests.dll
  System.Net.WebSockets.Client -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.WebSockets.Client.dll
  System.IO.FileSystem.DriveInfo -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.FileSystem.DriveInfo.dll
  System.Linq.Queryable -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Linq.Queryable.dll
  System.IO.MemoryMappedFiles -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.MemoryMappedFiles.dll
  System.Net.Ping -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.Ping.dll
  System.ComponentModel.TypeConverter -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ComponentModel.TypeConverter.dll
  System.Net.NameResolution -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Net.NameResolution.dll
  Microsoft.Packaging.Tools.Trimming.Tasks -> e:\src\ms\dotnet\standard\bin\AnyOS.AnyCPU.Debug\Microsoft.Packaging.Tools.Trimming.Tasks\Microsoft.Packaging.Tools.Trimming.Task
  s.dll
  System.Diagnostics.TraceSource -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.TraceSource.dll
  System.Globalization.Extensions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Globalization.Extensions.dll
  System.IO.Compression.ZipFile -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.Compression.ZipFile.dll
  System.Linq -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Linq.dll
  System.Xml.Linq -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Xml.Linq.dll
  System.Diagnostics.Tracing -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.Tracing.dll
  System.ServiceModel.Web -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.ServiceModel.Web.dll
  System.Dynamic.Runtime -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Dynamic.Runtime.dll
  System.IO.FileSystem.Watcher -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.FileSystem.Watcher.dll
  System.Web -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Web.dll
  System.Net -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Net.dll
  System.Diagnostics.Process -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.Process.dll
  System.IO -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.dll
  System.Diagnostics.TextWriterTraceListener -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.TextWriterTraceListener.dll
  System.Linq.Expressions -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Linq.Expressions.dll
  System.Numerics -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Numerics.dll
  System.Data -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Data.dll
  System.Diagnostics.Debug -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.Debug.dll
  System.Globalization.Calendars -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Globalization.Calendars.dll
  System.Diagnostics.Contracts -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.Contracts.dll
  System.IO.IsolatedStorage -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.IsolatedStorage.dll
  System.AppContext -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.AppContext.dll
  System.Drawing -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Drawing.dll
  System.ComponentModel.EventBasedAsync -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ComponentModel.EventBasedAsync.dll
  System.Diagnostics.StackTrace -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.StackTrace.dll
  System.ComponentModel.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ComponentModel.Primitives.dll
  System.IO.FileSystem -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.IO.FileSystem.dll
  System.Security.Principal.Windows -> e:\src\ms\dotnet\standard\bin\ref\extensions\System.Security.Principal.Windows.dll
  System.Collections.NonGeneric -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Collections.NonGeneric.dll
  System.Data.Common -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Data.Common.dll
  System.Windows -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Windows.dll
  System.Globalization -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Globalization.dll
  System.Collections.Concurrent -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Collections.Concurrent.dll
  System.ComponentModel -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.ComponentModel.dll
  System.Runtime.Serialization -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Runtime.Serialization.dll
  System.Diagnostics.Tools -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.Tools.dll
  System.Xml -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Xml.dll
  System.Collections -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Collections.dll
  System.Security.AccessControl -> e:\src\ms\dotnet\standard\bin\ref\extensions\System.Security.AccessControl.dll
  System.Diagnostics.FileVersionInfo -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Diagnostics.FileVersionInfo.dll
  System.CodeDom -> e:\src\ms\dotnet\standard\bin\ref\extensions\System.CodeDom.dll
  System.Console -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Console.dll
  System.Core -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Core.dll
  Microsoft.Win32.Registry -> e:\src\ms\dotnet\standard\bin\ref\extensions\Microsoft.Win32.Registry.dll
  System.ComponentModel.Composition -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.ComponentModel.Composition.dll
  System.Xml.Serialization -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Xml.Serialization.dll
  System.Collections.Specialized -> e:\src\ms\dotnet\standard\bin\shims\netstandard\System.Collections.Specialized.dll
  System.Transactions -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.Transactions.dll
  Microsoft.Win32.Primitives -> e:\src\ms\dotnet\standard\bin\shims\netstandard\Microsoft.Win32.Primitives.dll
  System.IO.Compression.FileSystem -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.IO.Compression.FileSystem.dll
  System -> e:\src\ms\dotnet\standard\bin\shims\netfx\System.dll
  mscorlib -> e:\src\ms\dotnet\standard\bin\shims\netfx\mscorlib.dll
  Microsoft.Packaging.Tools.Trimming.Tasks -> e:\src\ms\dotnet\standard\bin\AnyOS.AnyCPU.Debug\Microsoft.Packaging.Tools.Trimming.Tasks\net451\Microsoft.Packaging.Tools.Trimmi
  ng.Tasks.dll
  Microsoft.Packaging.Tools.Trimming -> e:\src\ms\dotnet\standard\bin/packages/Debug/specs/Microsoft.Packaging.Tools.Trimming.nuspec
  NETStandard.Library -> e:\src\ms\dotnet\standard\bin/packages/Debug/specs/NETStandard.Library.nuspec
Command execution succeeded.
```

--- a/config.json
+++ b/config.json
@@ -123,7 +123,7 @@
         "packages": {
           "description": "Builds the NuGet packages.",
           "settings": {
-            "Project": "netstandard/package.builds",
+            "Project": "packages.builds",
             "FilterToOSGroup":"default",
             "MsBuildLogging":"/flp:v=normal;LogFile=build-packages.log"
           }

